src/scrabble.js

```diff
@@ -1,6 +1,73 @@
 class Scrabble {
   // Write your implementation here
+  constructor(word) {
+    this.word = word;
+    this.points = 0;
+  }
 
-}
+  score(word) {
+    if (this.word === "" || this.word === " ") {
+      this.points += 0;
+      return 0;
+    }
+
+    if (this.word === " \t\n") {
+      this.points += 0;
+      return 0;
+    }
+
+    if (this.word === null) {
+      this.points += 0;
+      return 0;
+    }
 
-module.exports = Scrabble
+    for (let i = 0; i < this.word.length; i++) {
+      const pointOne = ["a", "e", "i", "o", "u", "l", "n", "r", "s", "t"];
+      const pointTwo = ["d", "g"];
+      const pointThree = ["b", "c", "m", "p"];
+      const pointFour = ["f", "h", "v", "w", "y"];
+      const pointFive = ["k"];
+      const pointEight = ["j", "x"];
+      const pointTen = ["q", "z"];
+
+      if (
```

Comment:
for lines 33 to 68 an `else {this.points += 0}` is needed otherwise illegal/unidentified characters will not be scored and could break the system